### PR TITLE
adding network flag to AJAX requests

### PR DESCRIPTION
**Pull request by dashifen**
This fixes WP-CFM's network problems, I think.  Essentially, it adds a hidden input field on the settings page which has a value of 1 when it's the network admin and 0 when it's not.  Then, this value is included in all of the AJAX requests and used in the `includes/class-options.php` file to identify a network bundle.

Take a look at things and let me know if you see anything obviously problematic.  I'm going to test things a bit more locally, but if this seems to fix things, then we can send it up to ForumOne as a PR for the main package.  And, if they don't want to use it, we can keep using our patched version.

Also, the a11y in this plugin is rock bottom.  Inputs without labels, no `aria` attributes, nothing.  😒

### CODE REVIEW

File: assets/js/admin.js [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
         // Load
         $.post(ajaxurl, {
-            action: 'wpcfm_load',
-            compare_env: compare_env
+            'action': 'wpcfm_load',
+            'compare_env': compare_env,
```
Comment (dashifen) [inline assets/js/admin.js:7]: I was just adding single quotes here - the rest of the file uses them for object properties, these two lines didn't.

**Review by thatdevgirl — APPROVED** [review]
Wow... that's really all this needs for multisite support? It seems so simple. I hope they accept it!

**Comment by dashifen** [comment]
well, this fixes the network bundles, i.e. bundles that specify network options and are created from the network admin pages.  this doesn't create a bundle-type that will work for all sites on a network.  that's what I'm calling a "global" bundle and I've got those being created, but now I have to get them so that they can be read into any or all sites.

**Comment by dashifen** [comment]
[Correct WP-CFM for easier operation on multisite networks.](https://trello.com/c/U71Dn3VA/399-correct-wp-cfm-for-easier-operation-on-multisite-networks)

**Pull request merged**